src/pages/404/notFound.module.css

```diff
@@ -26,86 +25,55 @@ p {
   margin: 0;
 }
 
-.sadImg {
+.error{
```

Comment:
@vedikaag99 Please update the className accordingly

Reply:
done